components/style/themes/default.less

```
@@ -341,6 +341,7 @@
 // Modal
 // --
 @modal-header-bg: @component-background;
+@modal-footer-bg: @component-background;
```

```suggestion
@modal-footer-bg: tranparent;
```

To keep the original style.